src/seedu/addressbook/data/person/Person.java

```diff
@@ -8,7 +8,7 @@
  * Represents a Person in the address book.
  * Guarantees: details are present and not null, field values are validated.
  */
-public class Person implements ReadOnlyPerson {
+public class Person implements ReadOnlyPerson, Printable {
```

Comment:
Is it really necessary to implement Printable in the Person class. Could have just implemented getPrintableString(Printable...printables) right?